build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -267,9 +267,10 @@ public static BuildTargetResult GenerateMSbuildPropsFile(BuildTargetContext c)
             msbuildProps.AppendLine($"    <HostVersion>{hostVersion.LatestHostPolicyVersion.WithoutSuffix}</HostVersion>");
             msbuildProps.AppendLine($"    <HostResolverVersion>{hostVersion.LatestHostFxrVersion.WithoutSuffix}</HostResolverVersion>");
             msbuildProps.AppendLine($"    <HostPolicyVersion>{hostVersion.LatestHostVersion.WithoutSuffix}</HostPolicyVersion>");
-            msbuildProps.AppendLine($"    <BuildNumberMajor>{hostVersion.LatestHostBuildMajor}</BuildNumberMajor>");
-            msbuildProps.AppendLine($"    <BuildNumberMinor>{hostVersion.LatestHostBuildMinor}</BuildNumberMinor>");
-            msbuildProps.AppendLine($"    <PreReleaseLabel>{hostVersion.LatestHostPrerelease}</PreReleaseLabel>");
+            msbuildProps.AppendLine($"    <BuildNumberMajor></BuildNumberMajor>");
```

Comment:
This is used only for packaging. The build.sh takes care of baking in version info and rid info into the binary.